README.md

```diff
@@ -62,33 +62,39 @@ Thank you to all those who have translated asbplayer:
 If you are a non-English native, and would like to help translate asbplayer, please contact me on [Discord](https://discord.gg/ad7VAQru7m).
 
 ## Getting Started
+> [!NOTE]  
+> asbplayer can either be used purely as a tool for subtitle control or as a flashcard creation tool. If you want to strictly learn how to use asbplayer's subtitle features and are not interested in sentence or vocabulary mining, you can read about them under the [Subtitle setup and features](#subtitle-setup-and-features) header.

```

Comment:
We can make this shorter. e.g:
"asbplayer is both a subtitle control and flashcard creation tool. If are not interested in flashcards, and only want to use asbplayer's subtitle features, you can read about them under [Subtitle setup and features](#subtitle-setup-and-features)."

Side note: it's easier to understand what "them" means when "asbplayer's subtitle features" is closer to it.